jest.config.js

```diff
@@ -22,43 +22,38 @@
 
 const { resolve } = require('node:path')
 
+// Listed packages will be transformed with babel-jest
 // TODO: find a way to consolidate this in one place, with webpack.common.js
 const ignorePatterns = [
-	'(vue-material-design-icons)',
-	'(@juliushaertl)',
-	'(tributejs)',
-	'(@nextcloud/vue)',
-	'(splitpanes)',
-	'(string-length)',
-	'(strip-ansi)',
-	'(ansi-regex)',
-	'(char-regex)',
-	'(uuid)',
-	'(unist*)',
-	'(unified)',
-	'(bail)',
-	'(remark*)',
-	'(is-*)',
-	'(trough)',
-	'(vfile)',
-	'(mdast*)',
-	'(micromark)',
-	'(decode-named-character-reference)',
-	'(trim-lines)',
-	'(rehype*)',
-	'(hast-*)',
-	'(property-information)',
-	'(space-separated-tokens)',
-	'(comma-separated-tokens)',
-	'(web-namespaces)',
+	'bail',
+	'comma-separated-tokens',
+	'decode-named-character-reference',
+	'devlop',
+	'escape-string-regexp',
+	'hast-.*',
+	'is-.*',
+	'mdast-.*',
+	'micromark',
+	'property-information',
+	'rehype-.*',
+	'remark-.*',
+	'space-separated-tokens',
+	'trim-lines',
+	'trough',
+	'unified',
+	'unist-.*',
+	'vfile',
+	'vue-material-design-icons',
+	'web-namespaces',
 ]
 
 module.exports = {
 
 	// Allow tests in the src and in tests/unit folders
 	testMatch: ['<rootDir>/src/**/*.(spec|test).(ts|js)'],
+	// Transform packages from top-level and nested 'node_modules'
 	transformIgnorePatterns: [
-		'node_modules/(?!' + ignorePatterns.join('|') + '/)',
+		`<rootDir>/node_modules/(?!(?:.*\\/node_modules\\/)?(?:${ignorePatterns.join('|')}))`,
```

Comment:
This RegExp still has a false negative. Like
`/node_modules/ESM_PACKAGE/node_modules/CJS_PACKAGE/index.js`
will NOT be ignored because it is inside ESM, while it should be, it's CJS.

But I couldn't come up with anything better 😮‍💨

Reply:
Will it be a big problem? From what I understand, babel shouldn't transform CJS files (at least not heavily), depending on target version of babel, as CJS shoul be already compiled with suitable syntax

Reply:
It won't, it will just transform more than needed.

cjs is already compiled. But also not ignored in this specific case, so Jest will transform it with babel

Reply:
I'll raise an issue to track it